src/component/Component.js

```diff
@@ -121,7 +125,7 @@ export default class Component {
   addCustomField(key, value) {
     Object.defineProperty(this, key, {
       get: () => this.$privateStore.getItem(key),
-      set: value => this.$privateStore.setItem(key, value),
+      set: val => this.$privateStore.setItem(key, val),
```

Comment:
Is this necessary because of the redefining of the `value` parameter or is it just a shorthand?

Reply:
redefining, it was creating a shadow scope here

Reply:
Okay 😄